homeassistant/components/switcher_kis/quality_scale.yaml

```diff
@@ -0,0 +1,78 @@
+rules:
```

Comment:
`SwitcherBaseLightEntity` can get the
```py

        self._light_id = light_id
        self.control_result: bool | None = None
```
part to clean up the 2 child classes

Reply:
Fixed at https://github.com/home-assistant/core/pull/143607